README.md

```diff
@@ -33,6 +33,12 @@ To run all unit tests execute the following command:
 ./gradlew test
 ```
 
+### Functional tests
+Run:
+```bash
+./gradlew functionalTest
```

Comment:
Running functional tests requires configured instance of postgresql. Please add the missing information.